.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2
+
+jobs:
+  checkout:
+    working_directory: ~/openfisca-core
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - checkout
+
+      - save_cache:
+          key: v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - .
+
+  dependencies:
+    working_directory: ~/openfisca-core
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - restore_cache:
+          keys:
+            - v1-checkout-{{ .Environment.CIRCLE_SHA1 }}
+
+      - restore_cache:
```

Comment:
Isn't `v1-dependencies` set in this `dependencies` job?

Reply:
Circle warning: `No cache is found for key xxxxxx`

Reply:
> Isn't v1-dependencies set in this dependencies job?

Yes, the idea is that, it is cached the first time (no cache found), and it is restored subsequently.

Now, given cache is saved by commit, it won't ever be restored here, so I'll delete the step.